<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeBinToolClass</class>
 <widget class="QMainWindow" name="MergeBinToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MergeBinTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidgetFileInfo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>21</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWarningFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>189</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>注意：合并文件只包含文件头和文件数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputPath">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>221</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输出路径：</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFileName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>合并文件名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutPath">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>222</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMergeFile">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>250</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWarningOutput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>282</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>错误</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMerge">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>合并</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileHead">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>0</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>文件头</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MergeBinTool.qrc"/>
 </resources>
 <connections/>
</ui>
